README.md

```diff
@@ -58,6 +58,43 @@ subtle manual mistakes you can make when releasing from your terminal.
 
 You can release all the unpublished packages by running `release-plz release`.
 
+### Generate shell completions
+
+#### zsh
+To load completions in your current shell session:
+```sh
+$ autoload bashcompinit; bashcompinit
```

Comment:
This guy is a real troublemaker!

The documentation example for `kwctl` didn't work for me. 
```sh
$ source <(kwctl completions -s zsh)
```
So I used the only solution that I found is working. For me. At least.